sensing/autoware_pointcloud_preprocessor/launch/preprocessor.launch.xml

```diff
@@ -5,13 +5,51 @@
      "['/points_raw0', '/points_raw1', '/points_raw2', ...]"
      This syntax is also available from command line -->
   <arg name="input_points_raw_list" default="['/points_raw']" description="define as string_array"/>
-
   <arg name="output_points_raw" default="/points_raw/cropbox/filtered" description=""/>
   <arg name="tf_output_frame" default="base_link" description=""/>
+  <arg name="separate_concatenate_node_and_time_sync_node" default="false" description=""/>
+
+      <node_container pkg="rclcpp_components" exec="component_container" name="pointcloud_preprocessor_container" namespace="pointcloud_preprocessor">
+        <!-- Conditional: Check if concatenate and time sync nodes should be separated -->
+            <composable_node  unless="$(var separate_concatenate_node_and_time_sync_node)" pkg="autoware_pointcloud_preprocessor" plugin="autoware::pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent" name="sync_and_concatenate_filter" namespace="">
+                <remap from="~/input/twist" to="/sensing/vehicle_velocity_converter/twist_with_covariance" />
+                <remap from="output" to="/points_raw/concatenated" />
+                <param name="input_topics" value="$(var input_points_raw_list)" />
+                <param name="output_frame" value="$(var tf_output_frame)" />
+                <param name="approximate_sync" value="true" />
+                <param name="publish_synchronized_pointcloud" value="false" />
+                <param name="input_twist_topic_type" value="twist" />
+            </composable_node>
+
+        <!-- Separate components for time sync and concatenation -->
+        <composable_node if="$(var separate_concatenate_node_and_time_sync_node)" pkg="autoware_pointcloud_preprocessor" plugin="autoware::pointcloud_preprocessor::PointCloudDataSynchronizerComponent" name="synchronizer_filter" namespace="">
```

Comment:
So, in our meeting you explained that the `<group>` option is not allowed inside the `<node_container>` tag. Therefore, instead of using `node_container` and `composable_node` tag, you could use, `node_container` and `load_composable_node` tag. Like so:

```
<node_container pkg="pkg" exec="component_container_isolated" name="pointcloud_preprocessor_container" namespace=""/>
  <group if="$(var separate_concatenate_node_and_time_sync_node)">
    <load_composable_node target="pointcloud_preprocessor_container">
      <composable node pkg="autoware_pointcloud_preprocessor" plugin="autoware::pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent"
name="xxx" namespace="">
.
.
.
</composable_node>
<composable_node>
</composable_node>
</load_composable_node>
```